proto/regen/ecocredit/v1alpha1/tx.proto

```diff
@@ -136,11 +144,19 @@ message MsgSendRequest {
     string tradable_units = 2
         [ (gogoproto.moretags) = "yaml:\"tradable_units\"" ];
 
-    // retired_units are the units of credits in this issuance that are
-    // effectively retired by the issuer on receipt. Decimal values are
+    // retired_units are the units of credits in this transfer that are
+    // effectively retired by the recipient on receipt. Decimal values are
     // acceptable within the precision returned by Query/Precision.
     string retired_units = 3
         [ (gogoproto.moretags) = "yaml:\"retired_units\"" ];
+
+    // retirement_location is the location of the beneficiary or buyer of the
+    // retired credits. This must be provided if retired_units is positive. It
+    // is a string of the form
+    // <country-code>[-<sub-national-code>[-<postal-code>]], with the first two
```

Comment:
This needs to be updated to describe the space character and postal code format (64 char alphanumeric)